locales/ja/validation-inline.php

```diff
@@ -12,116 +12,116 @@
+    'boolean'              => 'trueまたはfalseを指定してください。',
+    'confirmed'            => '確認の内容が一致しません。',
+    'date'                 => '有効な日付ではありません。',
+    'date_equals'          => ':date 同じ日付を指定してください。',
+    'date_format'          => ':format の形と一致しません。',
+    'different'            => ':other とは異なる必要があります。',
+    'digits'               => ':digits桁でなければなりません。',
+    'digits_between'       => ':min ~ :max桁でなければなりません。',
+    'dimensions'           => '画像の寸法が無効です。',
+    'distinct'             => '値が重複しています。',
+    'email'                => '無効なメールアドレスです。',
+    'ends_with'            => '次のいずれかで終わる必要があります: :values',
     'exists'               => '選択した値が無効です。',
-    'file'                 => '内容はファイルである必要があります。',
-    'filled'               => 'このフィールドには値が必要です。',
+    'file'                 => 'ファイルである必要があります。',
+    'filled'               => '値がありません。',
     'gt'                   => [
-        'array'   => 'The content must have more than :value items.',
-        'file'    => 'The file size must be greater than :value kilobytes.',
-        'numeric' => 'The value must be greater than :value.',
-        'string'  => 'The string must be greater than :value characters.',
+        'array'   => ':value 個より多くい必要があります。',
+        'file'    => 'ファイルサイズが :value KBより大きい必要があります。',
+        'numeric' => ':value より大きい必要があります。',
+        'string'  => ':value文字より多い必要があります。',
     ],
     'gte'                  => [
-        'array'   => 'The content must have :value items or more.',
-        'file'    => 'The file size must be greater than or equal :value kilobytes.',
-        'numeric' => 'The value must be greater than or equal :value.',
-        'string'  => 'The string must be greater than or equal :value characters.',
+        'array'   => ':value 個以上でなければなりません。',
+        'file'    => 'ファイルサイズが :value KB以上でなければなりません。',
+        'numeric' => ':value 以上でなければなりません。',
+        'string'  => ':value文字以上必要です。',
     ],
-    'image'                => 'これは画像でなければなりません。',
+    'image'                => '画像でなければなりません。',
     'in'                   => '選択した値が無効です。',
-    'in_array'             => 'この値は:otherには存在しません。',
-    'integer'              => 'これは整数でなければなりません。',
+    'in_array'             => 'この値は:otherに存在しません。',
+    'integer'              => '数字でなければなりません。',
     'ip'                   => '有効なIPアドレスである必要があります。',
     'ipv4'                 => '有効なIPv4アドレスである必要があります。',
     'ipv6'                 => '有効なIPv6アドレスである必要があります。',
-    'json'                 => 'これは有効なJSON文字列である必要があります。',
+    'json'                 => '有効なJSON文字列である必要があります。',
     'lt'                   => [
-        'array'   => 'The content must have less than :value items.',
-        'file'    => 'The file size must be less than :value kilobytes.',
-        'numeric' => 'The value must be less than :value.',
-        'string'  => 'The string must be less than :value characters.',
+        'array'   => ':value 個より少ない必要があります。'
```

Comment:
need ','

Reply:
Sorry for that. Fixed in 34fa86dcf9ba74bd4052826fdc230521a8a376bd